kubernetes/yamls/yaydoc/yaydoc-deployment.yml

```diff
@@ -0,0 +1,28 @@
+kind: Deployment
+apiVersion: apps/v1beta1
+metadata:
+  name: yaydoc
+  namespace: web
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: yaydoc
+    spec:
+      containers:
+      - name: yaydoc
+        image: ujjwalbhardwaj/yaydoc:latest
```

Comment:
Yeah, and I told you that I was getting an error while doing. I tried a lot of things but couldn't find any solution.